### PR TITLE
fix StorageClass filtering for add capacity modal

### PR DESCRIPTION
**Pull request by SanjalKatiyar**
BZ: https://bugzilla.redhat.com/show_bug.cgi?id=2213183

**Before:**
Screenshot 2023-09-19 at 7 07 01 PM

**After:**
Screenshot 2023-09-19 at 7 07 44 PM

### CODE REVIEW

**Comment by SanjalKatiyar** [comment]
@GowthamShanmugam PTAL.

File: packages/odf/utils/ocs.ts [hunk 1]
```diff
@@ -195,9 +195,11 @@ export const getDeviceSetCount = (pvCount: number, replica: number): number =>
...
 
 export const filterSC = (sc: StorageClassResourceKind) =>
-  !OCS_PROVISIONERS.some((ocsProvisioner: string) =>
-    sc?.provisioner?.includes(ocsProvisioner)
-  );
+  !sc
+    ? false
+    : !OCS_PROVISIONERS.some((ocsProvisioner: string) =>
+        sc?.provisioner?.includes(ocsProvisioner)
+      );
```
Comment (GowthamShanmugam) [inline packages/odf/utils/ocs.ts:202]:
```suggestion
  !!sc &&  !OCS_PROVISIONERS.some((ocsProvisioner: string) =>
        sc?.provisioner?.includes(ocsProvisioner)
   );
```

**Review by GowthamShanmugam — APPROVED** [review]

**Pull request merged**